RecoEgamma/Configuration/python/RecoEgamma_EventContent_cff.py

```diff
@@ -164,7 +164,10 @@
 )
 
 # mods for HGCAL
-_phase2_hgcal_RecoEgamma_tokeep = [ 'keep *_ecalDrivenGsfElectronCores_*_*', 'keep *_ecalDrivenGsfElectrons_*_*' ]
+_phase2_hgcal_RecoEgamma_tokeep = [ 'keep *_ecalDrivenGsfElectronCores*_*_*',
+                                    'keep *_ecalDrivenGsfElectrons*_*_*',
```

Comment:
we do not allow wildcards in the module names in keep statements for reco objects